src/templates/layouts/vamcSystem/index.tsx

```diff
@@ -0,0 +1,191 @@
+import { VamcSystem as FormattedVamcSystem } from '@/types/formatted/vamcSystem'
+import { MediaImage } from '@/templates/common/mediaImage'
+import { ContentFooter } from '@/templates/common/contentFooter'
+import { LovellStaticPropsResource } from '@/lib/drupal/lovell/types'
+import { useEffect } from 'react'
+import { SideNavMenu } from '@/types/formatted/sideNav'
+// import { LovellSwitcher } from '@/templates/components/lovellSwitcher'
+// import { TopTasks } from '@/templates/components/topTasks'
+// import { FacilityListing } from '@/templates/components/facilityListing'
+// import { MainButtons } from '@/templates/components/mainButtons'
+// import { ListOfLinkTeasers } from '@/templates/components/listOfLinkTeasers'
+// import { NewsStoryTeaser } from '@/templates/components/newsStoryTeaser'
+// import { EventTeaser } from '@/templates/components/eventTeaser'
+// import { SocialLinks } from '@/templates/common/socialLinks'
+
+// Allows additions to window object without overwriting global type
+interface customWindow extends Window {
+  sideNav?: SideNavMenu
+}
+declare const window: customWindow
+
+export function VamcSystem({
+  title,
+  introText,
+  image,
+  administration,
+  fieldRelatedLinks,
+  path,
+  menu,
+  // vamcEhrSystem,
+  // mainFacilities,
+  // newsStoryTeasersFeatured,
+  // eventTeasersFeatured,
+  // eventTeasersAll,
+  // lovellVariant,
+  // lovellSwitchPath,
+}: LovellStaticPropsResource<FormattedVamcSystem>) {
+  // Populate the side nav data for the side nav widget to fill in
+  // Note: The side nav widget is in a separate app in the static-pages bundle
+  useEffect(() => {
+    window.sideNav = menu
+  })
+  
+  // TODO: It looks like the reason the image is taller than in the original page is
+  // because the image file is actually different. It has a taller intrinsic height.
+  // Here is an example from `/washington-dc-health-care/`:
+  // Old:
+  // https://s3-us-gov-west-1.amazonaws.com/content.www.va.gov/img/styles/7_2_medium_thumbnail/public/2021-08/Washington%20VA%20Medical%20Center.jpg
+  // New:
+  // https://dsva-vagov-staging-cms-files.s3.us-gov-west-1.amazonaws.com/styles/2_1_large/public/2021-08/Washington%20VA%20Medical%20Center.jpg
```

Comment:
I'm not ready for criticisms to be lobbied against this image. I'll fix it in another pull request

Reply:
And if anyone has ideas about my comments here, I'd welcome any insights

Reply:
Ah, yeah. There's Some Stuff™ that happens with images to generate different styles. You've just gotta use the correct one. I forget what the details off the top of my head, but you can find an example here (similar to what you've got below): https://github.com/department-of-veterans-affairs/next-build/blob/main/src/templates/components/imageAndStaticMap/index.tsx#L16-L20